share/eds/eds_test.go

```diff
@@ -175,6 +176,34 @@ func TestReadEDSContentIntegrityMismatch(t *testing.T) {
 	require.ErrorContains(t, err, "share: content integrity mismatch: imported root")
 }
 
+func BenchmarkReadWriteEDS(b *testing.B) {
+	ctx, cancel := context.WithCancel(context.Background())
+	b.Cleanup(cancel)
+	for originalDataWidth := 4; originalDataWidth <= 64; originalDataWidth *= 2 {
```

Comment:
I'd add either a comment here, or preferably a docstring on the benchmark what these criteria are and why we are using these specific values.

Reply:
done